src/libs/OptionsListUtils.js

```diff
@@ -230,7 +231,7 @@ function createOption(personalDetailList, report, {
         ? (hasMultipleParticipants && lastActorDetails
             ? `${lastActorDetails.displayName}: `
             : '')
-        + Str.htmlDecode(report.lastMessageText)
+        + (isLastMessageAttachment ? `[${Localize.translateLocal('common.attachment')}]` : Str.htmlDecode(report.lastMessageText))
```

Comment:
Could we please simplify this ternary? lets not nest it multi levels.

Reply:
This should be okay. Any suggestion?
```diff
    const lastMessageTextFromReport = isLastMessageAttachment ? `[${Localize.translateLocal('common.attachment')}]` : Str.htmlDecode(report.lastMessageText)
    const lastMessageText = report
        ? (hasMultipleParticipants && lastActorDetails
            ? `${lastActorDetails.displayName}: `
            : '')
        + (lastMessageTextFromReport)
        : '';
```

Reply:
Let's break the concatenation into the next line.
```
const lastMessageTextFromReport = isLastMessageAttachment ? `[${Localize.translateLocal('common.attachment')}]` : Str.htmlDecode(report.lastMessageText);
 let lastMessageText = report && hasMultipleParticipants && lastActorDetails
            ? `${lastActorDetails.displayName}: `
            : '';
            
     lastMessageText  +=  report? lastMessageTextFromReport : '';
```